res/drawable/issue_event_closed.xml

```diff
@@ -0,0 +1,15 @@
+<vector
+    xmlns:android="http://schemas.android.com/apk/res/android"
+    android:width="24dp"
+    android:height="24dp"
+    android:viewportHeight="24.0"
+    android:viewportWidth="24.0">
+    <path
+        android:fillAlpha="1"
+        android:fillColor="#e65858"
```

Comment:
When I tried to use `@color` in vector drawables in past it didn't work. But, now I see a warning saying that the generated pngs might not have the correct colors. Perhaps they improved that since I last tried. I'll try that and test how it will look on API < 21.

I wonder if it would work when enabling compat vector drawables too. I use them successfully in my own app (there were some issues with proguard) but never tried using color references. If they indeed work then perhaps it's time to replace all the pngs :wink: